src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestBuilder.cs

```diff
@@ -197,41 +201,72 @@ await remote.CommitUpdatesAsync(
         return description.ToString();
     }
 
-    public async Task<string> GenerateCodeFlowPRTitleAsync(
-        SubscriptionUpdateWorkItem update,
-        string targetBranch)
+    public string GenerateCodeFlowPRTitle(
+        string targetBranch,
+        List<string> repoNames)
     {
-        return await CreateTitleWithRepositories($"[{targetBranch}] Source code changes from ", [update.SubscriptionId]);
+        return GeneratePRTitle($"[{targetBranch}] Source code changes from ", repoNames);
```

Comment:
Let's also change it to this:

```suggestion
        return GeneratePRTitle($"[{targetBranch}] Source code updates from ", repoNames);
```